README.md

```diff
@@ -171,15 +171,16 @@ that required paid sponsorship upon exceeding a free limit.
-| `--memory`      | no override | Override default image memory configuration (size in megabytes or `auto`<sup>1</sup>)                                                                           |
-| `--dir`         |             | `--dir` arguments to pass to `tart run`, can be specified multiple times                                                                                        |
-| `--disk`        |             | `--disk` arguments to pass to `tart run`, can be specified multiple times                                                                                       |
-| `--auto-prune`  | true        | Whether to enable or disable the Tart's auto-pruning mechanism (sets the `TART_NO_AUTO_PRUNE` environment variable for Tart command invocations under the hood) |
-| `--allow-image` |             | only allow running images that match the given [doublestar](https://github.com/bmatcuk/doublestar)-compatible pattern, can be specified multiple times          |
+| Argument          | Default     | Description                                                                                                                                                     |
+|-------------------|-------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `--concurrency`   | 1           | Maximum number of concurrently running Tart VMs to calculate the `auto` resources                                                                               |
+| `--cpu`           | no override | Override default image CPU configuration (number of CPUs or `auto`<sup>1</sup>)                                                                                 |
+| `--memory`        | no override | Override default image memory configuration (size in megabytes or `auto`<sup>1</sup>)                                                                           |
+| `--dir`           |             | `--dir` arguments to pass to `tart run`, can be specified multiple times                                                                                        |
+| `--disk`          |             | `--disk` arguments to pass to `tart run`, can be specified multiple times                                                                                       |
+| `--auto-prune`    | true        | Whether to enable or disable the Tart's auto-pruning mechanism (sets the `TART_NO_AUTO_PRUNE` environment variable for Tart command invocations under the hood) |
+| `--allow-image`   |             | only allow running images that match the given [doublestar](https://github.com/bmatcuk/doublestar)-compatible pattern, can be specified multiple times          |
+| `--default-image` |             | A fallback tart image to use, in case the job does not specify one                                                                                              |
```

Comment:
```suggestion
| `--default-image` |             | A fallback Tart image to use, in case the job does not specify one                                                                                              |
```